pyglider/plotting.py

```diff
@@ -130,10 +267,10 @@ def timeseries_plots(fname, plottingyaml):
                 _log.info('cl %s', k)
                 if config['timeseries'][k] == 'True':
                     ax = axs[n]
-                    locator = mdates.AutoDateLocator(minticks=3, maxticks=7)
-                    formatter = mdates.ConciseDateFormatter(locator)
-                    ax.xaxis.set_major_locator(locator)
-                    ax.xaxis.set_major_formatter(formatter)
+           #         locator = mdates.AutoDateLocator(minticks=3, maxticks=7)
+           #         formatter = mdates.ConciseDateFormatter(locator)
```

Comment:
we probably want to keep this in here?

Reply:
Put it back in.

Reply:
Keeping this part of the code causes an out of index error in matplotlib/dates.py in the testing. I've looked at it, but I don't know enough about dates.py to troubleshoot it very effectively. Did you want me to just keep lines 270 and 271 or the entire block of code following those lines (and similar pieces of code elsewhere in plotting.py?

Reply:
We could probably figure out what is going on.  But I'm confused because this has been woking, so I'm not clear why it is broken now.

Reply:
Its because you didn't specify enough empty offsets.

_But_... why would you get rid of the offsets?  The dates can often be meaningless with out them

You guys also changed a bunch of things about the plots that were features not bugs.  You labeled all the ticks on each axes, which were explicitly not labeled in the original.  You changed the date formatting, etc.  I put substantial care into making the grid plots the way I want them.  I'm happy to take suggestion on them, but I'll revert the ones here.